### ISSUE TITLE
[api-extractor] Crash when tsconfig contains `importsNotUsedAsValues` option

### ISSUE THREAD

**Issue by AlCalzone** [microsoft/rushstack#1827]
## Please prefix the issue title with the project name i.e. [rush], [api-extractor] etc. ##

**Is this a feature or a bug?**

- [ ] Feature
- [x] Bug

**Please describe the actual behavior.**
The tool crashes with "unknown compiler option"
```
$ npx api-extractor run --local --verbose

api-extractor 7.7.13  - https://api-extractor.com/

Using configuration from C:\Repositories\node-zwave-js\api-extractor.json

Error: Error parsing tsconfig.json content: Unknown compiler option 'importsNotUsedAsValues'.
```

**What is the expected behavior?**
It should not crash.

**If this is a bug, please provide the tool version, Node.js version, and OS.**

* **Tool:** api-extractor
* **Tool Version:** 7.7.13
* **Node Version:** 12.1.0
  * **Is this a LTS version?** no
  * **Have you tested on a LTS version?** no
* **OS:** Windows 10

**Comment by SimenB** [comment]
Getting the same error. I thought `typescriptCompilerFolder` would have fixed it?

**Comment by octogonz** [comment]
If you read the docs, `typescriptCompilerFolder` changes the include paths, but it does not replace the compiler engine. API Extractor ships with a specific compiler engine that cannot easily be swapped out at runtime. Generally it's supposed to be the latest one - we should upgrade it.

**Comment by octogonz** [comment]
Fixed with PR https://github.com/microsoft/rushstack/pull/1983 which was released as API Extractor 7.9.0

**Comment by SimenB** [comment]
Aha, I just assumed it was the compiler used. Thanks for the fix!
